compiler/rustc_feature/src/unstable.rs

```diff
@@ -209,6 +209,8 @@ declare_features! (
     (internal, compiler_builtins, "1.13.0", None),
     /// Allows writing custom MIR
     (internal, custom_mir, "1.65.0", None),
+    /// Allows using `#[export]` which indicates that an item is exportable.
+    (internal, export, "CURRENT_RUSTC_VERSION", None),
```

Comment:
You can now add #139939 as the tracking issue (and move this to the section for features with a tracking issue).